libbeat/_meta/config.full.yml

```diff
@@ -139,13 +139,22 @@ output.elasticsearch:
   # These settings can be adjusted to load your own template or overwrite existing ones
 
   # Template name. By default the template name is beatname.
-  template.name: "beatname"
+  #template.name: "beatname"
```

Comment:
as it now also loads when commented how, we should have an option `template.enabled: false` to make it possible to not load the template